make/devices.cmake

```diff
@@ -851,7 +851,7 @@ function(ADD_FPGA_TARGET)
       )
   endif()
 
-  add_custom_target(${NAME}_eblif ALL DEPENDS ${OUT_EBLIF})
+  add_custom_target(${NAME}_eblif DEPENDS ${OUT_EBLIF})
```

Comment:
I understand this addresses `make` to succeed on a fresh checkout, but it is not clear this the desired default build target. I think we want the default build to generate more than the env. I'd be ok will this if we open an issue to discuss what the defualt build target should include.

Reply:
I'm working on another commit on this PR to add all targets for boards.

Reply:
https://github.com/SymbiFlow/symbiflow-arch-defs/issues/539